community-operators/prometheus/alertmanager.crd.yaml

```diff
@@ -1,6 +1,8 @@
+---
```

Comment:
nit: Is this part of some convention I'm unaware of?

Reply:
`---` is used to separate documents within a stream as per https://yaml.org/spec/1.1/#id857577. It is pretty idiomatic in many yaml files we use.